src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -899,10 +943,14 @@ const reducers = {
     isDashboardTableLoading: false,
   }),
   TOGGGLE_CHECKBOX_WITH_API: (state, item) => {
-    const updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
+    let updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(
```

Comment:
Codacy has a fix for the issue: [Unexpected newline after '('.](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8502145)
```suggestion
    let updatedCheckboxData1 = updateFilteredAPIDataIntoCheckBoxData(item.data.searchSubjects, facetSearchData,
```